source/client-side-encryption/tests/README.md

```diff
@@ -3683,3 +3683,60 @@ Run an aggregate operation on `db.csfle` with the following pipeline:
 ```
 
 Expect an exception to be thrown with a message containing the substring `Upgrade`.
+
+### 26. Custom AWS Credentials
+
+These tests require valid AWS credentials for the remote KMS provider via the secrets manager. These tests MUST NOT run
+inside an AWS environment that has the same credentials set in order to properly ensure the tests would fail using
+on-demand credentials.
```

Comment:
Suggest mentioning which keys we expect people to use:
```suggestion
These tests require valid AWS credentials for the remote KMS provider via the secrets manager (FLE_AWS_KEY and FLE_AWS_SECRET). These tests MUST NOT run
inside an AWS environment that has the same credentials set in order to properly ensure the tests would fail using
on-demand credentials.
```

Reply:
Updated.